test/python/transpiler/test_unitary_synthesis.py

```diff
@@ -570,6 +576,28 @@ def test_coupling_map_transpile(self, opt):
             )
         )
 
+    @combine(
+        opt_level=[0, 1, 2, 3],
+        bidirectional=[True, False],
+        dsc=(
+            "test natural_direction works with transpile using opt_level {opt_level} on"
+            "target with multiple 2q gates with bidirectional={bidirectional}"
```

Comment:
```suggestion
            "test natural_direction works with transpile using opt_level {opt_level} on"
            " target with multiple 2q gates with bidirectional={bidirectional}"
```
thoguh I wouldn't have bothered if I hadn't left another comment in the review, haha